### ISSUE TITLE
Can't find SD upscale.

### ISSUE THREAD

**Issue by Susanoo1337** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#136]
Since custom scripts are written specifically for Webui and Gradio I figured that's why they are not implemented in the plugin;
however if I were to make one suggestion, the one thing that stands in the way of this plugin enabling 100% of my workflow inside Photoshop is the lack of an SD upscale function. (Not the lossy upscalers in Extras)
In particular the way Ultimate SD Upscale script does it. (https://github.com/Coyote-A/ultimate-upscale-for-automatic1111)

If a custom implementation of that is possible at any point in the future, that would be excellent.

**Comment by Susanoo1337** [comment]
Absolutely: https://rentry.org/3dt34, not all of them but brings the point across I think, Ultimate SD upscale requires some more cherrypicking, as it tends to introduce some odd features at higher denoise scales but it also adds details to the image, which the Extra upscalers cannot do, hence why it would be ultra useful in photoshop.

My current workflow is to create a handful of SD upscales, pick the best ones based on singular features, and then blend them together in an image editor. Having that straight within the editor would cut out like 5 steps.

I.e. changing the canvas size in PS and then upscaling a new flattened layer to the new canvas dimensions would be the way to go.

**Comment by annasophiachristianahahn** [comment]
this would be amazing yes please!
